src/seedu/addressbook/AddressBook.java

```diff
@@ -1150,8 +1150,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String s, String prefix) {
+        return s.replaceFirst(prefix, "");
```

Comment:
Hi, Sorry I'm new to this so I'm messing up my pull requests.
I tested the above code and apparently if prefix is in the middle of string, it will be changed as well. I fixed this by using ^+prefix which matches the start of the string instead.